m4/ax_cxx_compile_stdcxx_11.m4

```diff
@@ -0,0 +1,142 @@
+# ============================================================================
```

Comment:
Indeed, the one [currently on the on-line AutoConf archive](https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx_11.html) doesn't match the one in the corresponding Debian 8 package.  I guess I should indeed update to the latest upstream, although it's functionally the same when it's used to check for C++11 and not anything newer.